examples/gnrc_tftp/tftp_client.c

```diff
@@ -59,7 +59,7 @@ static bool _tftp_client_start_cb(tftp_action_t action, tftp_mode_t mode,
     }
 
     /* display the action being performed */
-    printf("tftp_client: %s %s %s:%u\n", str_mode, str_action, file_name, *len);
+    printf("tftp_client: %s %s %s:%zu\n", str_mode, str_action, file_name, *len);
```

Comment:
`%zu` will not evaluate for several toolchains. When compiling for the `samr21-xpro` e.g. it will print `zu` instead of the proper value.